examples/contracts/generic_contract/src/generic.rs

```diff
@@ -128,18 +128,20 @@ mod tests {
         let app = App::<cw_multi_test::BasicApp<SvCustomMsg, SvCustomQuery>>::custom(|_, _, _| {});
         #[allow(clippy::type_complexity)]
         let code_id: CodeId<
-            SvCustomMsg,
-            SvCustomMsg,
-            SvCustomMsg,
-            SvCustomMsg,
-            SvCustomMsg,
-            SvCustomMsg,
-            SvCustomMsg,
-            SvCustomMsg,
-            SvCustomMsg,
-            SvCustomMsg,
-            SvCustomMsg,
-            String,
+            GenericContract<
```

Comment:
To be considered.

Move `concrete types` from `entry_points` macro to an attribute.

Make it mandatory in case the generics are defined on the Contract by emitting an error.

This way we will be able to generate alias for Contract with these types to be used later in the `write_api` and also in the tests f.e. by using it in the `CodeId`. This will be optional to be used and if user would like to use different concrete types in tests they will be able to.

Reply:
Should be resolved by https://github.com/CosmWasm/sylvia/issues/362